.github/workflows/docker-release-workflow.yaml

```diff
@@ -0,0 +1,58 @@
+name: Docker release reusable workflow
+
+on:
+  workflow_call:
+    inputs:
+      BUILD_ARGS:
+        required: true
+        type: string
+      DOCKER_FILE:
+        required: true
+        type: string
+    secrets:
+      DOCKERHUB_USERNAME:
+        required: true
+      DOCKERHUB_TOKEN:
+        required: true
+
+jobs:
+  release-docker:
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Get the Image tag
+        env:
+          TAG: ${{ github.ref_name }}
+        run: echo "FULL_IMAGE_TAG=concordium/${TAG}" >> $GITHUB_ENV
+
+      - name: Check if image exist
+        run: |
+          set +e
+          docker manifest inspect ${{ env.FULL_IMAGE_TAG }}
+          exitcode=$?
+          if [ $exitcode -eq "0" ]; then
+            echo "Error: ${{ env.FULL_IMAGE_TAG }} already exist"
+            exit 1
+          fi
+
+      - name: Clone repo
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+
+      - name: Build Docker Image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: ${{ inputs.DOCKER_FILE }}
+          tags: ${{ env.FULL_IMAGE_TAG }}
+          no-cache: true
+          push: false
+          build-args: |
+            ${{inputs.BUILD_ARGS}}
```

Comment:
Have you tested this with mulitple args?

Reply:
I have tested with:

```
rust_version=1.73.0
ghc_version=9.6.4
protoc_version=3.15.3
flatbuffers_tag=v22.12.06
nvm_sh_version=v0.37.2
cmake_version=3.25.1
```